### ISSUE TITLE
Think about removing riemann dependency

### ISSUE THREAD

**Issue by neshtea** [active-group/active-logger#9]
`active-logger` contains utilities for connecting and logging to a Riemann instance. While this may come in handy, most projects -- in general -- will not use this.  Since most of our Clojure projects (should) depend on `active-logger`, I find it unnecessary so include Riemann in the bill of materials of every project.

My suggestion would be to pull out the Riemann specific namespace (`active.clojure.logger.riemann`) and put it in it's own project as a kind of `active-logger` 'adapter'.  This might cause slight overhead, but since I only know of one project that uses the Riemann configuration/appender, it might be worth it,

Thoughts?

**Comment by cresh** [comment]
Good idea! I'd like to see it go the whole way, though, a full separation of api and backends, see #3.
